package.json

```diff
@@ -69,9 +71,12 @@
     "moment": "^2.20.1",
     "mousetrap": "^1.6.2",
     "music-metadata": "^3.5.3",
+    "node-fetch": "^2.6.0",
     "nuclear-core": "0.0.5",
     "nuclear-ui": "0.0.6",
     "numeral": "^2.0.6",
+    "once": "^1.4.0",
```

Comment:
Is this used anywhere? In any case it's better to use lodash's `_.once`

Reply:
mmmh i think it was stream related, and i finally remove it during a refacto